src/main/java/org/opentripplanner/routing/edgetype/StreetTransitLink.java

```diff
@@ -54,10 +54,6 @@ public LineString getGeometry() {
         return GeometryUtils.getGeometryFactory().createLineString(coordinates);
     }
 
-    public TraverseMode getMode() {
-        return TraverseMode.LEG_SWITCH;
-    }
-
     public String getName() {
         return "street transit link";
```

Comment:
Now this is returned in the walking instructions, should we try to generate a more descriptive name? Just return the name of the stop?

Reply:
If we wanted to preserve the same behavior, just ignoring the edge would be correct, but that may not be what we want.

Walk instructions like "turn right onto Oslo Central Station" does sound more useful to me. I pushed a commit with this change. I plan to work on moving the actual walk instructions over to the back end soon, and we may decide to further improve this. Maybe something like "you are now at Oslo Central Station".